### DISCUSSION TITLE
PikPak 挂载显示需要 captcha

### DISCUSSION THREAD

**Discussion by QYQSDTC** [AlistGo/alist#6495] [Q&A]
<img width="448" alt="CleanShot-YhhZaWEq@2x" src="https://github.com/alist-org/alist/assets/15608306/d285eb6e-32c2-4b45-a5cc-f3b957b6bc70">

PikPak和 PikPak 分享都显示 captcha_required

**Comment by li-peifeng** [comment]
xiaoya 已经修复了，等吧

**Reply by QYQSDTC** [reply]
xiaoya 太强了

**Reply by haojiaming1232** [reply]
咱就是说要多久呢

**Reply by li-peifeng** [reply]
已经修复了，可以自己编译下

**Comment by li-peifeng** [comment]
我的已经编译好了，需要的可以换一下镜像名
docker pull leolitaly/sweet-cloud:latest

**Reply by chafx9** [reply]
請問怎樣使用?

**Reply by li-peifeng** [reply]
你alist怎么用就怎么用啊， 只要换一下image的名称

**Reply by chafx9** [reply]
大大, 我是新手, 不明白你指什麼?我是發覺出現上面訊息後一直不能掛pikpak, 應該怎麽修改, 我打開alist都不知要在哪裡改?求指教

**Reply by li-peifeng** [reply]
https://alist.nn.ci/zh/guide/install/docker.html

停止或删除老的容器，然后运行命令
docker run -d --restart=unless-stopped -v /etc/alist:/opt/alist/data -p 5244:5244 -e PUID=0 -e PGID=0 -e UMASK=022 --name="alist" leolitaly/sweet-cloud:latest

**Reply by parano11d** [reply]
还是不太行，只能播放一半的进度条，后面的就播放不了了

**Reply by li-peifeng** [reply]
> 还是不太行，只能播放一半的进度条，后面的就播放不了了

我只是负责打包编译，其它无能为力，等官方修复了

**Reply by eric09230** [reply]
@li-peifeng 感謝大佬 已經可以成功掛載了

**Reply by li-peifeng** [reply]
> @li-peifeng 感謝大佬 已經可以成功掛載了

能用就好

**Reply by chafx9** [reply]
我想問如果我的alist是安裝在asus router, 應該如何去停止或删除老的容器，然后运行命令?
我不知在哪裡執行, 有大大可以幫忙教我嗎?謝謝

**Reply by li-peifeng** [reply]
> 我想問如果我的alist是安裝在asus router, 應該如何去停止或删除老的容器，然后运行命令?
> 
> 我不知在哪裡執行, 有大大可以幫忙教我嗎?謝謝

我不清楚你当初是如何安装的，router哪里运行命令我并不了解，你可以试着运行以下命令停止删除老的容器。
假如你的容器名为alist
docker stop alist
docker rm alist

**Reply by chafx9** [reply]
asus router有個app 可以安裝alist, 安裝完後我便用內網訪問alist 如192.168.1.1:5244, 便在瀏覽頁看到alist,
但沒有安裝docker, 所以我不懂怎樣去執行docker stop alist

**Reply by li-peifeng** [reply]
> asus router有個app 可以安裝alist, 安裝完後我便用內網訪問alist 如192.168.1.1:5244, 便在瀏覽頁看到alist,
> 
> 但沒有安裝docker, 所以我不懂怎樣去執行docker stop alist

这样啊，那可能不是用docker运行的， 现在官方已经更新到3.35.0了，您可以试着更新一下

**Reply by wearshoes** [reply]
感谢大佬编译 配置都不用改 直接替换镜像地址就行了

**Reply by haojiaming1232** [reply]
> 谢谢大佬的配置都不用改直接替换镜像地址就行了

咋替换的啊，能分享下不

**Reply by li-peifeng** [reply]
> > 谢谢大佬的配置都不用改直接替换镜像地址就行了
> 
> 
> 
> 咋替换的啊，能分享下不

你原来怎么安装的就怎么安装，只要替换image的名称就可以了

**Reply by clbin001** [reply]
docker模式，更新到了xiaoya最新版本3.35，pikpak终于可以挂载成功了，不再报需要Captcha的错误。但是仅仅可以看到目录和文件列表，如果点击任何图片或视频文件，都无法播放。请问安装了leolitaly/sweet-cloud:latest镜像的朋友，可以正常播放图片和视频文件吗？

**Comment by clbin001** [comment]
docker模式，更新到了xiaoya最新版本3.35，pikpak终于可以挂载成功了，不再报需要Captcha的错误。但是仅仅可以看到目录和文件列表，如果点击任何图片或视频文件，都无法播放
